duckduckgo/iOS
# update experiments

Task/Issue URL:
Tech Design URL:
CC:

**Description**:

* Remove tab swticher experiment
* Remove ipad improvement experiments
* Update serp experiment to be based on locale 

**Steps to test this PR**:
1. Smoke test iPad improvements on a phone and an ipad
1. Smoke test tab switcher 
1. Using a real device (doesn't appear to work on simulator) set to to one of the expected locales, ensure allocated to one of the serp experiments.
1. Using simulator ensure not allocated to SERP experiment
1. Check region codes match list required list here: https://app.asana.com/0/72649045549333/1170000970522275
using reference: https://gist.github.com/jacobbubu/1836273



**Device Testing**:

* [x] iPhone - Any

**OS Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
